bin/iex

```diff
@@ -43,6 +43,14 @@ readlink_f () {
   fi
 }
 
+ERL="-noshell -user Elixir.IEx.CLI"
 SELF=$(readlink_f "$0")
 SCRIPT_PATH=$(dirname "$SELF")
-exec "$SCRIPT_PATH"/elixir --no-halt --erl "-noshell -user Elixir.IEx.CLI" +iex "$@"
+
+if [ -z "$HOME" ]; then
+  :
+else
+  ERL="-kernel shell_history_path \"$HOME/.elixir-history\" $ERL"
+fi
```

Comment:
@dimitarvp yes, please do send a PR. Let's go with HOMEDRIVE/HOMEPATH for now for consistency then. My only question is if you have to quote the `%DIR%` variable somehow. Or at least test that the feature works if your user directory has a space on it.